storage/src/main/java/org/apache/uniffle/storage/common/LocalStorage.java

```diff
@@ -47,6 +47,7 @@ public class LocalStorage extends AbstractStorage {
 
   private final long diskCapacity;
   private volatile long diskFree;
```

Comment:
I think the original variables naming is not appropriate.

Reply:
Do you have any good ideas?

Reply:
The same, maybe diskFree -> diskAvailableBytes ?

Reply:
ok for me. WDYT @jerqi @zuston

Reply:
Ok for me.